docs/guides/hapi-plugin.rst

```diff
@@ -4,9 +4,13 @@ Using Hoodie as hapi plugin
 Here is an example usage of Hoodie as a hapi plugin:
 
 .. code:: js
+
     var Hapi = require('hapi')
     var hoodie = require('hoodie').register
-
+    var PouchDB = require('pouchdb-core')
+    .plugin(require('pouchdb-mapreduce'))
+    .plugin(require('pouchdb-adapter-memory'))
```

Comment:
could you indent lines 11 & 12 with two spaces? That will make it more clear that these lines belong to what ` require('pouchdb-core')` returns